src/item_contents.cpp

```diff
@@ -322,9 +324,11 @@ ret_val<item_pocket *> item_contents::find_pocket_for( const item &it,
         ret_val<item_pocket::contain_code> ret_contain = pocket.can_contain( it );
         if( ret_contain.success() ) {
             return ret_val<item_pocket *>::make_success( &pocket, ret_contain.str() );
+        } else {
+            msg = ret_contain.str();
         }
     }
-    return ret;
+    return ret_val<item_pocket *>::make_failure( null_pocket, _( msg ) );
```

Comment:
Translation doesn't work like this, the string being translated needs to be available before compile time, which it is not here.

Reply:
I forgot about how it works, thanks, fixed it.